### DISCUSSION TITLE
guide-R/manipulation/formater-nombre

### DISCUSSION THREAD

**Discussion by giscus** [larmarange/guide-R#39] [Forum]
# guide-R/manipulation/formater-nombre

https://larmarange.github.io/guide-R/manipulation/formater-nombre.html

**Comment by WilliamZ98** [comment]
Bonjour monsieur,

Juste une remarque au niveau du point 15.2.8 label_ordinal() : vous avez écris <<scales::label_bytes() permets>>, au lieu de <<label_ordinal() permets>>

Respectueusement !

**Reply by larmarange** [reply]
Merci pour le signalement.

C'est maintenant corrigé.

Cdlt
